tests/Organizer/OrganizerLDProjectorTest.php

```diff
@@ -306,6 +307,40 @@ public function it_handles_address_updated($currentJson, $expectedJson): void
         $this->projector->handle($domainMessage);
     }
 
+    public function contactPointUpdatesDataProvider(): array
+    {
+        return [
+            'organizer without contact point' => [
+                'currentJson' => 'organizer.json',
+                'expectedJson' => 'organizer_with_updated_contactpoint.json',
+            ],
+        ];
+    }
+
+    /**
+     * @test
+     * @dataProvider contactPointUpdatesDataProvider
```

Comment:
A dataprovider with just one case is a bit weird. So better to use the json file names internally inside the test method itself.